Sources/Private/Experimental/Animations/LayerProperty.swift

```diff
@@ -51,60 +57,70 @@ extension LayerProperty {
   static var position: LayerProperty<CGPoint> {
     .init(
       caLayerKeypath: "transform.translation",
+      defaultValue: CGPoint(x: 0, y: 0),
```

Comment:
Interesting idea. Some immediate thoughts:
 - we can't create a collection of these since they all have different generic types (unless we create a type erased wrapped), so a test would probably have to test each one manually
  - the snapshot tests fail if you change these to an incorrect value, so we do currently have a layer of tests validating that they work as expected